engine/lib/notification.php

```diff
@@ -645,9 +645,6 @@ function elgg_send_email($from, $to, $subject, $body, array $params = null) {
 	$message->setSubject($subject);
 	$message->setBody($body);
 
-	foreach ($result['headers'] as $headerName => $headerValue) {
```

Comment:
this should not be removed, as plugin hooks can return header lines which need to be set